docs/dm/zh_CN/configuration/configuration.md

```diff
@@ -93,47 +110,23 @@ syncers:                                            # syncer 处理单元运行
 ```
 mysql-instances:
     -
-    ​    config:                                    # instance-id 对应的上游数据库配置 
-    ​        host: "192.168.199.118"
-    ​        port: 4306
-    ​        user: "root"
-    ​        password: "1234"                       # 需要使用 dmctl 加密的密码，具体说明见 [dmctl 使用手册]
-    ​    instance-id: "instance118-4306"            # MySQL Instance ID，对应上游 MySQL 实例，不允许配置 DM-master 的集群拓扑配置外的 mysql-instance 
-
-    ​    meta:                                      # 下游数据库的 checkpoint 不存在时 binlog 同步开始的位置; 如果 checkpoint 存在则没有作用 
-    ​        binlog-name: binlog-00001
+    ​    source-id: "mysql-replica-01"           # 上游实例或者复制组 ID，参考 inventory.ini 的 source_id 或者 dm-master.toml 的 source-id 配置
+    ​    meta:                                   # 下游数据库的 checkpoint 不存在时 binlog 同步开始的位置; 如果 checkpoint 存在则以 checkpoint 为准
+    ​        binlog-name: binlog-00001           # task-mode 为 full 和 all 的时候不需要设置
```

Comment:
`# task-mode 为 full 和 all 的时候不需要设置`
这个注释位置会让用户有误解为“只不需要设置binlog-name”？

Reply:
I think we can write like this because there are too many uncertainties, if there are other needs, we should suggest to customization this feature with us